app/src/components/collectionLanes/collectionLanes.tsx

```diff
@@ -6,38 +6,67 @@ import {
   Heading,
   Spacer,
   Link,
+  Icon,
 } from "@nypl/design-system-react-components";
 import styles from "./Collectionlanes.module.css";
 import useBreakpoints from "../../hooks/useBreakpoints";
 import CollectionCard from "../cards/collectionCard";
 import CollectionDataType from "../../types/CollectionDataType";
 import { CollectionCardModel } from "../../models/collectionCard";
 import DCSimpleGrid from "../dcSimpleGrid/dcSimpleGrid";
+import { headerBreakpoints } from "@/src/utils/breakpoints";
 
 const CollectionLanes = ({ lanesWithNumItems, seeMoreLink }) => {
   const { isLargerThanLargeTablet } = useBreakpoints();
   const lanes = lanesWithNumItems?.map((lane, key) => (
     <Box className={styles.lane} data-testid={lane.slug} mt="xxl" key={key}>
       <Flex alignItems="baseline">
-        <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+        <Heading
+          id={`row-heading-${lane.slug}`}
+          sx={{
+            [`@media screen and (min-width: ${headerBreakpoints.smTablet}px)`]:
+              {
+                width: "90%",
+              },
+          }}
+          level="h2"
+          size="heading3"
+        >
           {lane.name}
         </Heading>
         <Spacer />
         <Link
           id={`row-see-more-${lane.slug}`}
-          type="standalone"
```

Comment:
Why doesn't this work anymore?

Reply:
I wanted to explicitly prevent the arrow wrapping onto another line and the necessary CSS made it easier to just recreate from scratch